ui/push-health/TestFailure.jsx

```diff
@@ -132,34 +139,37 @@ class TestFailure extends React.PureComponent {
             />
           ))}
         </div>
-        {!!logLines.length &&
-          logLines.map(logLine => (
-            <Row
-              className="small text-monospace mt-2 ml-3"
-              key={logLine.line_number}
+        {!!logLines.length && (
+          <div>
```

Comment:
I moved this to the top, so it's easier to toggle if there's a lot of data..... like there is for crashes...  :)